justfile

```diff
@@ -5,6 +5,7 @@ export RUST_BACKTRACE := "1"
 install_tools: 
 	cargo install sqlx-cli
 	cargo install cargo-pretty-test
+	cargo install cargo-watch
```

Comment:
Like the name suggests, it allows to autoreload the app in dev when the source code changes